src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -324,6 +326,70 @@ def fuzz(self,
                                 artifact_prefix, additional_args, extra_env)
 
 
+class FuchsiaQemuLibFuzzerRunner(new_process.ProcessRunner, LibFuzzerCommon):
+  """libFuzzer runner (when Fuchsia is the target platform)."""
+
+  def __init__(self, executable_path, default_args=None):
+    if (not environment.get_value('FUCHSIA_PKEY_PATH')) or (
+        not environment.get_value('FUCHSIA_PORTNUM')):
+      raise fuchsia.errors.FuchsiaConfigError(
+          'FUCHSIA_PKEY_PATH and/or FUCHSIA_PORTNUM was not set')
+    self.ssh_root = [
+        'ssh', '-i',
+        environment.get_value('FUCHSIA_PKEY_PATH'), '-o',
+        'StrictHostKeyChecking no', 'localhost', '-o',
+        'UserKnownHostsFile=/dev/null', '-p',
+        str(environment.get_value('FUCHSIA_PORTNUM'))
```

Comment:
Can we `#yapf: disable` here, too?  Ideally, we'd have something like:
```
  'ssh', 
  '-i', environment.get_value('FUCHSIA_PKEY_PATH'), 
  '-o', 'StrictHostKeyChecking no',
  '-o', 'UserKnownHostsFile=/dev/null', 
  '-p', str(environment.get_value('FUCHSIA_PORTNUM'))
  'localhost', 
```

Reply:
done